src/py-opentimelineio/opentimelineio/core/_core_utils.py

```diff
@@ -177,7 +177,9 @@ def __deepcopy__(self, memo):
 
                 # Hide the method frm Sphinx doc.
                 # See https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#info-field-lists  # noqa
-                getattr(mapClass, name).__doc__ += '\n\n:meta private:'
+                getattr(mapClass, name).__doc__ = "{}\n\n:meta private:".format(
+                    getattr(mapClass, name).__doc__ or ""
+                )
```

Comment:
```suggestion
                # Add comment here to explain why we need to check for None
                if getattr(mapClass, name).__doc__ is not None:
                    getattr(mapClass, name).__doc__ += "\n\n:meta private:"
```
I think it would be clearer if we checked for `None` specifically. Also a little comment would be great so that we don't forget why sometimes it's None.

Reply:
You're right! Sphinx won't ever be run to generate documentation on an embedded python, so there is no need to add the tag if \_\_doc\_\_ is None.